### PR TITLE
fix end method in VoiceBroadcast

### PR DESCRIPTION
**Pull request by thomasxd24**
this.client is a ClientVoiceManager and thus its this.client.broadcasts  instead of this.client.voice.broascasts

**Please describe the changes this PR makes and why it should be merged:**

**Status**
- [x] Code changes have been tested against the Discord API, or there are no code changes
- [x] I know how to update typings and have done so, or typings don't need updating

**Semantic versioning classification:**  
- [ ] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: src/client/voice/VoiceBroadcast.js [hunk 1]
```diff
@@ -67,8 +67,8 @@ class VoiceBroadcast extends EventEmitter {
    */
   end() {
     for (const dispatcher of this.dispatchers) this.delete(dispatcher);
-    const index = this.client.voice.broadcasts.indexOf(this);
-    if (index !== -1) this.client.voice.broadcasts.splice(index, 1);
+    const index = this.client.broadcasts.indexOf(this);
```
Comment (vladfrangu) [inline src/client/voice/VoiceBroadcast.js]: Can you rename the client variable to voiceManager, including updating the jsdocs in the constructor? That way it's correctly showing what it is

Comment (SpaceEEC) [reply src/client/voice/VoiceBroadcast.js]: Or pass client when constructing `VoiceBroadcast`:
https://github.com/discordjs/discord.js/blob/f479afca8212e00208799793aac0286553669106/src/client/voice/ClientVoiceManager.js#L40

Comment (thomasxd24) [reply src/client/voice/VoiceBroadcast.js]: i think its better to pass the client

Comment (thomasxd24) [reply src/client/voice/VoiceBroadcast.js]: just change the pull to change the this.client 
`const broadcast = new VoiceBroadcast(this.client); `

**Pull request merged**